src/clj/athenaeum/session/redis.clj

```diff
@@ -0,0 +1,33 @@
+(ns athenaeum.session.redis
+  (:require [taoensso.carmine :as car]
+            [athenaeum.config :as config]))
+
+(defonce server-conn (atom nil))
+
+(defn set-conn-opts
+  []
+  (reset! server-conn {:pool {}, :spec {:host {:redis-host @config/config}}}))
+
+(defmacro wcar*
+  [& body]
+  `(car/wcar @server-conn ~@body))
+
+(defn set-key
+  [key value]
+  (wcar* (car/set key value)
+         (car/expire key 3600)))
```

Comment:
This function should probably accept the TTL as an optional argument. Only the session-specific namespace should know about the TTL, and the TTL itself should be in config.